spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -141,7 +141,7 @@ class VoipPush < PushCertificate; end
       class Passbook < Certificate; end
 
       # ApplePay certificate
-      class ApplePay < Certificate; end
```

Comment:
🚨 

This is not used anywhere else?
Could this be a breaking change for existing users of spaceship?

Reply:
Should I change it back to `ApplePay`? I don't remember why I changed it in the first place.

Reply:
I am not really sure. You probably changed it as it is a better description of what it actually is, right? Can we add backwards compat via an alias?

Reply:
added.